src/Lykke.Service.HFT.Services/MatchingEngineAdapter.cs

```diff
@@ -105,7 +107,7 @@ public class MatchingEngineAdapter : IMatchingEngineAdapter
                 CancelPreviousOrders = cancelPreviousOrders,
                 Volume = Math.Abs(volume),
                 OrderAction = orderAction.ToMeOrderAction(),
-                Fee = await GetLimitOrderFee(clientId, assetPairId, orderAction)
+                Fees = new[] { await GetLimitOrderFee(clientId, assetPairId, orderAction) }
             };
 
             var response = await _matchingEngineClient.PlaceLimitOrderAsync(order);
```

Comment:
Not related to the commit. You can use a cancellation token to avoid endless waiting for response from ME.